src/modules/emotes/channel-emotes.js

```diff
@@ -22,10 +23,10 @@ class ChannelEmotes extends AbstractEmotes {
         return provider;
     }
 
-    updateChannelEmotes({urlTemplate, emotes}) {
+    updateChannelEmotes({channelEmotes}) {
         this.emotes.clear();
 
-        emotes.forEach(({id, code, imageType, channel}) => (
+        channelEmotes.forEach(({id, code, imageType, channel}) => (
```

Comment:
channel is now `user` which is a dictionary. it is optional and won't exist on channelEmotes